Social meta tags

jonrogoff opened the pull request:
Resolves #51

wontruefree commented:
Would it be reasonable to test something like this?

wontruefree commented:
posibly the component where things are not entirely static?

jonrogoff commented:
I needed to push it somewhere other than localhost in order to test it.  As I feared, the social media parsers do not pick up these tags.  Looks like we will need to find a pre-processing solution and we aren't ready to do that just yet.

Once we have image content and a tagline, I will add something static so at least we have something global that looks a bit better.  Until then, the only change here that matters is the text correction on the faq page.

wzrad commented:
@jonrogoff I think it'd be worth rolling back the helmet stuff, adding the static meta tags into index.ejs, and pushing this through.

jonrogoff commented:
Helmet stuff rolled back and static meta tags added to index.ejs.
Added image assets (in /assets).
Added appropriate images to og:image and twitter:image.
Added favicon.
Replaced FontAwesome and text banner with banner image.

The images work locally, but not on staging.  Relative address problem, deployment problem or s3 bucket problem?

jonrogoff commented:
Fixed image deployment problem, but fix fails validation.

wzrad commented on package.json:
```diff
@@ -23,7 +23,10 @@
   },
   "dependencies": {
     "color": "^1.0.3",
+    "copy-webpack-plugin": "^4.0.1",
```
We don't need these dependencies in the app, so let's move them into dev dependencies. I don't think the yarn.lock discriminates between prod / dev dependencies, so you can just move them in the package.json. You can install something as a dev dependency w/ `yarn add -D <dep>`.

wzrad commented on src/navigation/header.js:
```diff
@@ -1,18 +1,16 @@
       <RouteLink {...rules.logoLink} to='/'>
-        <FontAwesome name='institution' />
-        <span>Legislated</span>
+        <img src='/assets/legislated_headerlogo-01.png' alt='Legislated' height='40' width='280' />
```
What do you think about normalizing the names of the assets `logo.png`, `logo_favicon.png`. Having the dimensions in there, for example, will break / cease making sense if we swap it out with something new. I also think it's correspondingly redundant and confusing to have "legislated" and "-09" in our asset names.

For the title asset, I think we should use the version without text, and render the text ourselves. Typography looks really bad when its rendered in an image on screens with non-matching densities (like phones).

jonrogoff commented:
I made the recommended changes.  I'll let you and Natalie argue over image asset names and the right font for our logo.  I don't particularly care one way or another.

I'm not sure how to persist the assets outside the repo such that they will work properly across all environments.  If you have a solution for that, fine by me.
